core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -93,21 +93,34 @@ private MetadataColumns() {}
           "_commit_snapshot_id",
           Types.LongType.get(),
           "Commit snapshot ID");
+  public static final NestedField ROW_ID =
+      NestedField.optional(
+          Integer.MAX_VALUE - 107, "_row_id", Types.LongType.get(), "Row ID for row lineage");
+  public static final NestedField LAST_UPDATED_SEQUENCE_NUMBER =
+      NestedField.optional(
+          Integer.MAX_VALUE - 108,
+          "_last_updated_sequence_number",
+          Types.LongType.get(),
+          "Last updated sequence number for row lineage");
```

Comment:
"Sequence number when the row was last updated"? I don't know that we want to call out "row lineage" because people that see this may not know what that is.

Reply:
Fixed this!